Readability-readerable.js

```diff
@@ -30,7 +30,8 @@ var REGEXPS = {
 
 function isNodeVisible(node) {
   // Have to null-check node.style to deal with SVG and MathML nodes.
-  return (!node.style || node.style.display != "none") && !node.hasAttribute("hidden");
+  return (!node.style || node.style.display != "none") && !node.hasAttribute("hidden")
+		&& (!node.hasAttribute("aria-hidden") || node.getAttribute("aria-hidden") !== "true");
```

Comment:
Nitpick: Could you use spaces instead of tabs, please? I thought eslint covered this, but clearly I was wrong and need to fix that...

Also, could we just use `node.getAttribute("aria-hidden") != "true"` ? If the node doesn't have the attribute, the value will be null, so the comparison still works.

Reply:
Hm, odd. I just pulled your branch, and there's definitely actual tabs... It looks like they get picked up if I add the eslint rule `no-tabs`; perhaps the existing indenting rules are OK with them because they're indenting the second and later lines of the statement or something.

Reply:
It worked, thanks again!